pkg/kubelet/sysctl/whitelist.go

```diff
@@ -81,6 +81,9 @@ func NewWhitelist(patterns []string) (*patternWhitelist, error) {
 // respective namespaces with the host. This check is only possible for sysctls on
 // the static default whitelist, not those on the custom whitelist provided by the admin.
 func (w *patternWhitelist) validateSysctl(sysctl string, hostNet, hostIPC bool) error {
+	// The '/' separator is also accepted in place of a '.' in sysctl variable.
+	// More info: https://man7.org/linux/man-pages/man8/sysctl.8.html
+	sysctl = strings.Replace(sysctl, "/", ".", -1)
```

Comment:
this also doesn't seem to follow the logic described where the first separator determines how the remaining separators are handled

Reply:
Same as above.